src/services/search/facets.ts

```diff
@@ -13,16 +13,14 @@ export const fetchFacets = async ({
   fields?: string[]
   options?: ApiCallOptions
 } = {}): Promise<Facets> => {
-  const json = await postData({
+  return postData({
```

Comment:
BREAKING CHANGE

Reply:
yeah like I realized for both the type and this that what I actually want is `  return { ...facets, topFacets }` to keep the IF and add the new property...